soc/renesas/ra/ra4m1/soc.c

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
The OFS is used widely in the RA series.
It should place a `common` directory.

Reply:
@soburi, while it's true that the OFS is used in a lot of RA microcontrollers it seems the contents of the registers, and their memory location is different in pretty much every series. A quick check of the device datasheets shows that RA6 series has different OSF1 register contents than RA4 series, as does the RA0 series. Therefore, I think it is better to leave this where it is...

Reply:
If so, let's make only Kconfig common.